Tests/FoundationEssentialsTests/DataTests.swift

```diff
@@ -1645,8 +1645,9 @@ private final class DataTests {
         #endif
     }
 
-    @available(FoundationSpan 6.2, *)
-    @Test func inlineDataSpan() throws {
+    @Test
+    @available(macOS 10.14.4, iOS 12.2, watchOS 5.2, tvOS 12.2, *)
```

Comment:
You can just drop these availabilities entirely since the deployment target of the package is macOS 15-aligned releases